firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -76,6 +80,12 @@
   private static final String LOCKFILE_NAME_GENERATE_FID = "generatefid.lock";
   private static final String CHIME_FIREBASE_APP_NAME = "CHIME_ANDROID_SDK";
 
+  /* File used to read the version of Instance-ID(IID) SDK if it exists in the dependency tree.
+  This is needed to prevent incompatible versions of IID SDK to be used in parallel with FIS SDK. */
+  private static final String IID_SDK_PROP_FILE = "/firebase-iid.properties";
+  private static final String IID_VERSION_WITH_FIS = "20.1.1";
```

Comment:
Actually IID v20.1.0 already depended on FIS, but I assume we want to make sure they use the hotfixed version? right?

If that is the case, please add an appropriate comment, example:

// FirebaseInstanceId v20.1.0 already depended on FirebaseInstallations for identifier creation, but had an issue of possible silent failures. Thus, v20.1.1 or later is recommended.